easybuild/easyconfigs/b/binutils/binutils-2.28-GCCcore-6.3.0.eb

```diff
@@ -18,6 +18,7 @@ builddependencies = [
 ]
 
 dependencies = [
+    # see https://github.com/easybuilders/easybuild-easyblocks/issues/1350
```

Comment:
@geimer Maybe make this:
```python
# zlib is a runtime dep to avoid that it gets embedded in libbfd.so,
# see https://github.com/easybuilders/easybuild-easyblocks/issues/1350
```

Reply:
@boegel As discussed, IMO not embedding `zlib` should be the default, i.e., documenting why it is listed as a build dependency (which is already done) is more important than the other way around. But if you still want me to use your suggested wording, please let me know.

Reply:
@geimer I agree, but just posting a link without clarifying what it relates to is worse than not mentioning anything. :)

So, I would either remove the comment again, or add the extra line to clarify (I prefer the latter).

Reply:
@boegel Done.